Automattic/jetpack
# Contact Form: Wrap mail messages in HTML tags to avoid spam filters

This avoid's SpamAssassin's `HTML_MIME_NO_HTML_TAG` rule.

See #6114

#### Testing instructions:

1. From `master`, fill out a `[contact-form]` and copy the email source to http://spamcheck.postmarkapp.com/
1. Note the triggered `HTML_MIME_NO_HTML_TAG` rule
1. Check out this branch, and re-run the test. The rule should no longer be triggered.
